### PR TITLE
doc / new changes 

### PR DESCRIPTION
**Pull request by JeremyKono**
**Before submitting this PR, please make sure**:

- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

### CODE REVIEW

File: documentation/docs/connectors/balancer.md [hunk 1]
```diff
@@ -27,7 +24,7 @@ When creating Hummingbot Gateway, it picks up the Ethereum settings in the globa
 
 ![](/assets/img/connect-ethereum.gif)
 
-## Install Hummingbot Gateway
```
Comment (dennisocana) [inline documentation/docs/connectors/balancer.md:37]: Gateway is the name of our other project/software. So it should be in uppercase.

File: documentation/docs/connectors/beaxy.md [hunk 2]
```diff
@@ -17,7 +16,7 @@ Private keys and API keys are stored locally for the operation of the Hummingbot
 !!! tip
     For copying and pasting into Hummingbot, see [this page](https://hummingbot.zendesk.com/hc/en-us/articles/900004871203-Copy-and-paste-your-API-keys) for more instructions in our Hummingbot Help Center.
 
-### Creating Beaxy API Keys
+### Creating beaxy API keys
```
Comment (dennisocana) [inline documentation/docs/connectors/beaxy.md:19]: Beaxy is the name of exchange. This should be uppercase

File: documentation/docs/connectors/binance.md [hunk 3]
```diff
@@ -19,7 +16,7 @@ Private keys and API keys are stored locally for the operation of the Hummingbot
 !!! tip
     For copying and pasting into Hummingbot, see [this page](https://hummingbot.zendesk.com/hc/en-us/articles/900004871203-Copy-and-paste-your-API-keys) for more instructions in our Hummingbot Help Center.
 
-### Creating Binance API Keys
+### Creating binance API keys
```
Comment (dennisocana) [inline documentation/docs/connectors/binance.md:19]: "Binance"

File: documentation/docs/connectors/dydx-perpetual.md [hunk 4]
```diff
@@ -1,22 +1,17 @@
...
 
-### Supported Installation Types
+### Supported installation types
 
 Currently, this connector does not work on binary installation. Install Hummingbot via Docker or from source to use this connector.
 
 - Install via Docker: [Linux](/installation/linux/#install-via-docker) | [Windows](/installation/windows/#install-via-docker) | [macOS](/installation/mac/#install-via-docker) | [Raspberry Pi](/installation/raspberry/)
 - Install from source: [Linux](/installation/linux/#install-from-source) | [Windows](/installation/windows/#install-from-source) | [macOS](/installation/mac/#install-from-source) | [Raspberry Pi](/installation/raspberry/#install-from-source)
 
-### API Credentials and Stark Key
```
Comment (dennisocana) [inline documentation/docs/connectors/dydx-perpetual.md:19]: "API credentials"

File: documentation/docs/connectors/gate-io.md [hunk 5]
```diff
@@ -12,33 +8,24 @@ The "Gate ecosystem" consists of Gate.io, Wallet.io, HipoDeFi and Gatechain, all
 
 To use [Gate.io](https://www.gate.io/ref/4566709) connector for spot trading with Hummingbot, you need to create an acount and generate an API key.
 
-### Creating Gate io API Keys
+### Creating Gate io API keys
```
Comment (dennisocana) [inline documentation/docs/connectors/gate-io.md:11]: "Gate.io"

File: documentation/docs/connectors/perp-fi.md [hunk 6]
```diff
@@ -42,7 +26,7 @@ When creating Hummingbot Gateway, it picks up the Ethereum settings in the globa
 
 ![](/assets/img/connect-ethereum.gif)
 
-## Install Hummingbot Gateway
+## Install Hummingbot gateway
```
Comment (dennisocana) [inline documentation/docs/connectors/perp-fi.md:29]: "Gateway"

File: documentation/docs/connectors/perp-fi.md [hunk 7]
```diff
@@ -31,7 +15,7 @@ Perpetual Protocol is a decentralized perpetual contract trading protocol for ev
 - Some xDai asset in the wallet for gas
 - xUSDC - all trades are funded and settled in xUSDC. You can obtain xUSDC by depositing USDC and receiving the xUSDC equivalent on the Perpetual Finance exchange [here](https://perp.exchange)
 
-## Connecting to Perpetual Finance
+## Connecting to perpetual finance
```
Comment (dennisocana) [inline documentation/docs/connectors/perp-fi.md:18]: "Perpetual Finance"

File: documentation/docs/connectors/uniswap.md [hunk 8]
```diff
@@ -38,14 +28,11 @@ When creating Hummingbot Gateway, it picks up the Ethereum settings in the globa
 
 ![](/assets/img/connect-ethereum.gif)
 
-## Install Hummingbot Gateway
+## Install Hummingbot gateway
```
Comment (dennisocana) [inline documentation/docs/connectors/uniswap.md:31]: "Gateway"

File: documentation/docs/installation/gateway.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
-## What is Hummingbot Gateway?
+## What is Hummingbot gateway?
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:1]: "Gateway"

File: documentation/docs/installation/gateway.md [hunk 10]
```diff
@@ -15,7 +15,7 @@ Hummingbot Gateway is a API server that allows Hummingbot to connect to [protoco
 !!! note
     As this passphrase will be stored in unencrypted form in the Gateway environment, we recommend that you use a different password as the Hummingbot password, which is used to encrypt your API and private keys.
 
-## Install Gateway via Docker
+## Install gateway via Docker
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:18]: "Gateway"

File: documentation/docs/installation/gateway.md [hunk 11]
```diff
@@ -62,7 +62,7 @@ In case the port number used by Gateway is not set to the default value of `5000
 
 ![](/assets/img/config-gateway-api-port.gif)
 
-## ETH Gas Station
+## ETH gas station
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:65]: "ETH Gas Station" is a proper name

File: documentation/docs/installation/gateway.md [hunk 12]
```diff
@@ -86,11 +86,11 @@ If you choose not to enable ETH gas price, you would only need to set up the fix
 
 ![](/assets/img/ethgas-no.png)
 
-## Update Gateway via Docker
+## Update gateway via Docker
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:89]: "Gateway"

File: documentation/docs/installation/gateway.md [hunk 13]
```diff
@@ -86,11 +86,11 @@ If you choose not to enable ETH gas price, you would only need to set up the fix
...
 
 To update the docker container, run the `update-gateway.sh` script and and follow the prompt instructions. The update script allows you to stop and delete the running instance, and update the docker image if it is not the latest. Upon completion, it will automatically execute the create-gateway.sh script to create a new Gateway container instance.
 
-## Install Gateway via source
+## Install gateway from source
```
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:93]: "Gateway"

File: documentation/docs/strategies/arbitrage.md [hunk 14]
```diff
@@ -23,12 +18,12 @@ An opportunity arises when Hummingbot can buy on one exchange at a lower price a
 
 ![Figure 1: Hummingbot completes an arbitrage trade](/assets/img/arbitrage.png)
 
-## Prerequisites: Inventory
+## Prerequisites: inventory
```
Comment (dennisocana) [inline documentation/docs/strategies/arbitrage.md:21]: "Inventory" since the word is followed by a colon.

**Review by dennisocana — CHANGES_REQUESTED** [review]
See requested changes.

**Comment by JeremyKono** [comment]
Resolved all requested changes in the latest commit

File: documentation/docs/support.md [hunk 15]
```diff
@@ -14,7 +14,7 @@ When you sign up for our Discord, please check that the link you are accessing i
 **We will never initiate direct messages to our users.** If a random user imitating the core team or any of the community members sends you a DM don't hesitate to report it in our official Discord channel.
 
 !!! note
-    Currently, our Discord server is the only officially-supported online Hummingbot community. We do not maintain any other communities on Telegram, WeChat, Slack, or other applications. Please beware that any such communities (except for the official Hummingbot Discord) may be scams.
+    Currently, our Discord server and WeChat(Chinese) are the only officially-supported online Hummingbot community. We do not maintain any other communities on Telegram, Slack, or other applications. Please beware that any such communities (except for the official Hummingbot Discord and WeChat(Chinese)) may be scams.
```
Comment (dennisocana) [inline documentation/docs/support.md:17]: WeChat (Chinese)

Comment (JeremyKono) [reply documentation/docs/support.md]: Fixed

File: documentation/docs/support.md [hunk 16]
```diff
@@ -14,7 +14,7 @@ When you sign up for our Discord, please check that the link you are accessing i
 **We will never initiate direct messages to our users.** If a random user imitating the core team or any of the community members sends you a DM don't hesitate to report it in our official Discord channel.
 
 !!! note
-    Currently, our Discord server and WeChat(Chinese) are the only officially-supported online Hummingbot community. We do not maintain any other communities on Telegram, Slack, or other applications. Please beware that any such communities (except for the official Hummingbot Discord and WeChat(Chinese)) may be scams.
+    Currently, our Discord server and WeChat (Chinese) are the only officially-supported online Hummingbot community. We do not maintain any other communities on Telegram, Slack, or other applications. Please beware that any such communities (except for the official Hummingbot Discord and WeChat may be scams.
```
Comment (dennisocana) [inline documentation/docs/support.md:17]: Missing ) after the second "WeChat"

**Review by dennisocana — APPROVED** [review]

**Comment by dennisocana** [comment]
Thanks @JeremyKono, @cali-tomas, @glenntomas

**Pull request merged**